hack/dockerfile/install-binaries.sh

```diff
@@ -32,6 +41,14 @@ install_containerd() {
 	cp bin/ctr /usr/local/bin/docker-containerd-ctr
 }
 
+install_proxy() {
+	echo "Install docker-proxy version $LIBNETWORK_COMMIT"
+	git clone https://github.com/docker/libnetwork.git "$GOPATH/src/github.com/docker/libnetwork"
+	cd "$GOPATH/src/github.com/docker/libnetwork"
+	git checkout -q "$LIBNETWORK_COMMIT"
+	CGO_ENABLED=$CGO_ENABLED go build -ldflags="$PROXY_LDFALGS" -o /usr/local/bin/docker-proxy github.com/docker/libnetwork/cmd/proxy
```

Comment:
I have no idea how that bash stuff works :D I thought you export it or use it inline, just variables not used by spawned command, I might be wrong, though.